scripts/debian/postinst.in

```diff
@@ -41,3 +41,34 @@ case "$1" in
 		fi
 	;;
 esac
+
+# Based off what debhelper dh_systemd_enable/13.3.4 would have added
+# ref: https://www.debian.org/doc/manuals/debmake-doc/ch05.en.html#debhelper
+
+if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
+        # This will only remove masks created by d-s-h on package removal.
+        deb-systemd-helper unmask 'falco.service' >/dev/null || true
+
+        # was-enabled defaults to true, so new installations run enable.
+        if deb-systemd-helper --quiet was-enabled 'falco.service'; then
+                # Enables the unit on first installation, creates new
+                # symlinks on upgrades if the unit file has changed.
+                deb-systemd-helper enable 'falco.service' >/dev/null || true
+        else
+                # Update the statefile to add new symlinks (if any), which need to be
+                # cleaned up on purge. Also remove old symlinks.
+                deb-systemd-helper update-state 'falco.service' >/dev/null || true
+        fi
+fi
+
+if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
```

Comment:
Isn't this the same if statement as above? Can we merge them?

Reply:
I don't think this debhelper-generated block is functionally the same as the configure block that already exists;

I did a little reading here to find out more about how these maintainer-script conditions are called: https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html#summary-of-ways-maintainer-scripts-are-called

* configure: all files and dependencies unpacked and configured
* abort-upgrade: all files unpacked, dependencies configured in a unexpected state, attempt to correct or configure again later -- I believe I have ran into situations like these when `apt-get` states that a package was not installed properly and continues to (attempt to) install it during subsequent `apt-get` runs
* abort-deconfigure: package still needs configuring, attempt to correct or configure later
* abort-remove: if package is an unexpected configured state, abort removal and keep it "installed"

Because of this, I don't think this is equivalent to the code above.
For the code above that is associated with installing/adding the kernel module to DKMS, it makes sense to associate these steps for after the falco package has been installed and configured properly. (The other states implies that there are problems during the installation/configuration of the package.)

Other than that, visually, I wanted to keep the auto-generated debhelper contents separate.